runsc/boot/loader.go

```diff
@@ -319,6 +322,17 @@ func New(args Args) (*Loader, error) {
 		return nil, fmt.Errorf("creating pod mount hints: %v", err)
 	}
 
+	var stdioFDs []int
+	newfd := startingStdioFD
+	for _, fd := range args.StdioFDs {
+		err := syscall.Dup3(fd, newfd, 0)
+		if err != nil {
+			return nil, fmt.Errorf("dup3 of stdioFDs failed: %v", err)
+		}
+		stdioFDs = append(stdioFDs, newfd)
```

Comment:
You can close `fd` here since it's not going to be used anymore.

Reply:
Ah right. I had the false impression that these FDs are passed through unix domain sockets through urpc but they are actually inherited through cmd.Start(). Will close them, thanks.